tasks/cat2.yml

```diff
@@ -143,28 +132,579 @@
-  template: src=audit.rules.j2 dest=/etc/audit/audit.rules owner=root group=root mode=0640 backup=yes
-  tags: [ 'cat2' , 'V-38580' , 'V-38527' , 'V-38525' , 'V-38522' , 'V-38552' , 'V-38550' , 'V-38557' , 'V-38556' , 'V-38554' , 'V-38559' , 'V-38558' , 'V-38536' , 'V-38540' , 'V-38541' , 'V-38543' , 'V-38547' , 'V-38530' , 'V-38578' , 'V-38575' , 'auditd' ]
-  notify: restart auditd
+- name: "MEDIUM | V-38481 | PATCH | System security patches and updates must be installed and up-to-date."
+  yum:
+      name: '*'
+      state: latest
+      update_cache: yes
+  tags:
+      - cat2
+      - medium
+      - V-38481
+      - yum
+      - updates
+      - patch
+
+- name: "MEDIUM | V-38483 | AUDIT | The system package management tool must cryptographically verify the authenticity of system software packages during installation."
+  shell: find /etc/yum{.conf,.repos.d/} -exec grep -ls '^gpgcheck=0' {} \;
```

Comment:
Per the standard I think this check should only check in `/etc/yum.conf`. There is a CAT 3 check V-38487 that is the same check but for the repo files in `/etc/yum.repos.d/`. Goes the same for the V-38483 patch item below.